src/fs.rs

```diff
@@ -963,6 +967,33 @@ mod tests {
         assert_eq!(m, mime::APPLICATION_OCTET_STREAM);
     }
 
+    #[test]
+    fn test_if_modified_since_ignored() {
+        assert!(NamedFile::open("test--").is_err());
+        let mut file = NamedFile::open("Cargo.toml")
+            .unwrap()
+            .set_cpu_pool(CpuPool::new(1));
+        {
```

Comment:
What is purpose of this block? and block below with assignment of file

Reply:
I don't know. I copied the code from another unit test, and I can see in the other unit tests which related to static file, they all contain these 2 blocks. If you think it is meaningless, I can try to remove all these blocks in each unit test.

What's more, I also don't understand the purpose of `.set_cpu_pool(CpuPool::new(1))`, could you please explain to me? Thank you.

Reply:
Yeah, I think you can remove it as this test only wants to test your fix anyway.

As for setting custom cpu pool, I'm not sure, but I see no harm it anyway as you gonna avoid using implicit global one